File: solutions/python/pangram/1/pangram.py
def is_pangram(sentence):
    abc = 'abcdefghijklmnopqrstuvwxyz'
    norm_sentence = sentence.lower()
    letter_count = 0
    
    for letter in abc:
        if norm_sentence.count(letter) > 0:
            letter_count += 1

    return letter_count >= 26

File: solutions/javascript/pangram/1/pangram.js
//
// This is only a SKELETON file for the 'Pangram' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const isPangram = (sentence) => {
  const normSentence = sentence.toLowerCase().replace(/[^a-z]/g, '');
  return new Set(normSentence).size >= 26;
};
